docs/types.rst

```diff
@@ -102,6 +102,10 @@ and to send Ether (in units of wei) to an address using the ``send`` function:
     to make safe Ether transfers, always check the return value of ``send`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+* ``tranfser``
+
+Transfer operates the same way as ``send``, with the exception that it will cause a exception if the transfer has failed.
```

Comment:
Please first explain transfer (because it is the recommended way) and then state that for special needs - if you want to recover a failed transfer - there is also send.